src/main/java/com/eprosima/fastdds/idl/templates/SwigCMake.stg

```diff
@@ -75,13 +75,18 @@ INCLUDE(\${SWIG_USE_FILE})
 SET(CMAKE_SWIG_FLAGS "")
 
 FIND_PACKAGE(PythonLibs REQUIRED)
+FIND_PACKAGE(PythonInterp REQUIRED)
```

Comment:
According to the [CMake documentation](https://cmake.org/cmake/help/latest/module/FindPythonLibs.html), both of these packages are deprecated in favor of FindPython3 since version 3.12. If we want to still use them, their order should be swapped:

```suggestion
FIND_PACKAGE(PythonInterp REQUIRED)
FIND_PACKAGE(PythonLibs REQUIRED)
```